opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -819,15 +820,21 @@ mod tests {
 
     impl LogProcessor for FirstProcessor {
         fn emit(&self, data: &mut LogData) {
-            // add attribute
-            data.record.attributes.get_or_insert(vec![]).push((
+            // Ensure attributes is initialized if it is empty
```

Comment:
can we modify the test processor to do what a user would do, instead of directly relying on implementation internals like GrowableArray?